### PR TITLE
feature: Add signing (hmac-sha256) of audit entries

### PR DESCRIPTION
**Pull request by jimlambrt**
This PR will add "signing" via hmac-sha256 of Boundary audit entries.
-  chore: Update go-eventlogger dependency
- feature: Add audit wrapper
- feature: Add hmac-sha256 of entries to hclogFormatterFilter
- feature: Add hmac-sha256 of entries to cloudEventsFormatterFilter

### CODE REVIEW

**Review by jefferai — APPROVED** [review]

File: internal/observability/event/cloudevents_formatter_node.go [hunk 1]
```diff
@@ -67,6 +70,24 @@ func newCloudEventsFormatterFilter(source *url.URL, format cloudevents.Format, o
...
+	}
+	f.l.Lock()
+	defer f.l.Unlock()
+	h, err := newSigner(context.Background(), w, nil, nil)
+	if err != nil {
+		return err
+	}
+	f.Signer = cloudevents.Signer(h)
+	return nil
+
```
Comment (louisruch) [inline internal/observability/event/cloudevents_formatter_node.go:88]: `make fmt`

File: sdk/go.sum [hunk 2]
```diff
@@ -237,6 +237,8 @@ github.com/hashicorp/errwrap v1.1.0 h1:OxrOeh75EUXMY8TBjag2fzXGZ40LB6IKw45YeGUDY
 github.com/hashicorp/errwrap v1.1.0/go.mod h1:YH+1FKiLXxHSkmPseP+kNlulaMuP3n2brvKWEqk/Jc4=
 github.com/hashicorp/eventlogger v0.1.0 h1:S6xc4gZVzewuDUP4R4Ngko419h/CGDuV/b4ADL3XLik=
```
Comment (louisruch) [inline sdk/go.sum:238]: `go mod tidy`

File: go.sum [hunk 3]
```diff
@@ -458,6 +458,8 @@ github.com/hashicorp/errwrap v1.1.0 h1:OxrOeh75EUXMY8TBjag2fzXGZ40LB6IKw45YeGUDY
 github.com/hashicorp/errwrap v1.1.0/go.mod h1:YH+1FKiLXxHSkmPseP+kNlulaMuP3n2brvKWEqk/Jc4=
 github.com/hashicorp/eventlogger v0.1.0 h1:S6xc4gZVzewuDUP4R4Ngko419h/CGDuV/b4ADL3XLik=
```
Comment (louisruch) [inline go.sum:459]: `go mod tidy`

File: internal/observability/event/cloudevents_formatter_node_test.go [hunk 4]
```diff
@@ -343,3 +344,42 @@ func TestNode_Process(t *testing.T) {
 		})
 	}
 }
+
+func Test_cloudEventsFormatter_Rotate(t *testing.T) {
+
```
Comment (louisruch) [inline internal/observability/event/cloudevents_formatter_node_test.go:349]: `make fmt`

**Review by louisruch — DISMISSED** [review]
Just need to run `make fmt` and `go mod tidy` also under the sdk

File: go.sum (hunk 3 above)
Comment (jimlambrt) [reply go.sum:459]: ty

File: internal/observability/event/cloudevents_formatter_node_test.go (hunk 4 above)
Comment (jimlambrt) [reply internal/observability/event/cloudevents_formatter_node_test.go:349]: ty!

File: internal/observability/event/cloudevents_formatter_node.go (hunk 1 above)
Comment (jimlambrt) [reply internal/observability/event/cloudevents_formatter_node.go:88]: ty!

File: sdk/go.sum (hunk 2 above)
Comment (jimlambrt) [reply sdk/go.sum:238]: ty!

**Pull request merged**